src/DynamoCore/Graph/Nodes/TypeLoadData.cs

```diff
@@ -97,6 +98,67 @@ public TypeLoadData(Type typeIn)
                 .SelectMany(x => x.PortTypes);
         }
 
+        public TypeLoadData (Type typeIn, ICollection<Attribute> attributes)
+        {
+            Type = typeIn;
+
+            ObsoleteMessage = string.Join(
+                "\n",
+                attributes.Where(x=>x is ObsoleteAttribute)
+                    .Cast<ObsoleteAttribute>()
+                    .Select(x => string.IsNullOrEmpty(x.Message) ? "Obsolete" : x.Message));
+
+            IsDeprecated = attributes.Where(x=>x is NodeDeprecatedAttribute).Cast<NodeDeprecatedAttribute>().Any();
```

Comment:
```suggestion
            IsDeprecated = attributes.OfType<NodeDeprecatedAttribute>().Any();
```
Also if you just need the count, no need to cast them?

Reply:
no....... not sure why i did that 😬 probably copy-pasted this from one where i needed something on the attribute